x-pack/solutions/search/plugins/serverless_search/public/plugin.ts

```diff
@@ -211,3 +221,16 @@ export class ServerlessSearchPlugin
 
   public stop() {}
 }
+
+// possible onboarding tokens now: 'general' | 'vector' | 'timeseries' for serverless, 'vectorsearch' or 'search' for hosted
+
+function onboardingTokenToWorkflowId(token: string | undefined | null): WorkflowId {
+  switch (token) {
+    case 'vector':
+      return 'vector';
+    case 'vectorsearch':
+      return 'vector';
+    default:
+      return 'default';
```

Comment:
is there a reason why is`semantic` type is not included in this list?

Reply:
Wasn't listed in the cloud code so would never be populated, but added it now